patterns/2-structured/maturity-model.md

```diff
@@ -153,60 +134,55 @@ One of the possible reasons for introducing InnerSource into organisations can b
 
-InnerSource projects need a means for self assessment. Metrics can be one aspect to facilitate this assessment. Also, in organisations with a mature InnerSource adoption level we expect adoption of the method to be tracked based on clear, agreed upon metrics.
+InnerSource projects need a means for self-assessment. Metrics can be one aspect to facilitate this assessment. Also, in organizations with a mature InnerSource adoption level, we expect the adoption of the method to be tracked based on clear, agreed-upon metrics.
 
 * MP-0: No existing monitoring policies at any level in the organization.
 * MP-1: Metrics are important for certain teams, and they start using them in an isolated way.
-* MP-2: There is a strategy at the organizational level with respect to metrics that help to validate specific policies across the organization. This monitoring policy exists at the level of some InnerSource projects.
-* MP-3: There are clear guidelines, recommendations, and trainings about the use of metrics with certain infrastructure provided by the organization. This works at both levels: InnerSource program to understand the general InnerSource adoption within the organization, and at the level of InnerSource projects.
+* MP-2: There is a strategy at the organizational level concerning metrics that help to validate specific policies across the organization. This monitoring policy exists at the level of some InnerSource projects.
+* MP-3: There are clear guidelines, recommendations, and training about the use of metrics with certain infrastructures provided by the organization. This works at both levels: InnerSource program to understand the general InnerSource adoption within the organization and at the level of InnerSource projects.
 
-**Support and Maintenance**
+#### Support and Maintenance
 
-Not only should feature development be owned by InnerSource teams - support and maintenance is also part of the teams core tasks.
+Not only should feature development be owned by InnerSource teams - support and maintenance is also part of the team's core tasks.
 
-* SM-0: Support given by the core dev or support team. A business contract guaranties the support. There is no knowledge about the product outside the team.
+* SM-0: Support given by the core dev or support team. A business contract guarantees support. There is no knowledge about the product outside the team.
 * SM-1: There are rules and regulations to formalize the support on the product, given by a dedicated supporting team.
-* SM-2: Support for InnerSource contributions is formalized through InnerSource patterns like [30 Day Warranty](./30-day-warranty.md) or [Service vs. Library](./service-vs-library.md).
-* SM-3: There are rules and regulations to formalize the support on the product, given by a mature community.
+* SM-2: Support for InnerSource contributions is formalized through InnerSource patterns like 30-Day[Warranty](./30-day-warranty.md) or [Service vs. Library](./service-vs-library.md).
```

Comment:
```suggestion
* SM-2: Support for InnerSource contributions is formalized through InnerSource patterns like [30 Day Warranty](./30-day-warranty.md) or [Service vs. Library](./service-vs-library.md).
```

The pattern name is "30 Day Warranty", so it is best to use exactly that phrase here too.